test/usersTest.js

```diff
@@ -213,11 +220,51 @@ describe('/users POST', () => {
               user.should.be.present;
               user.name.should.be.equal('Franco');
               user.password.should.not.be.equal('123456789');
+              dictum.chai(res);
+              done();
             });
-            dictum.chai(res);
-            done();
           });
         });
     });
   });
 });
+
+describe('/users GET', () => {
+  let request;
+  beforeEach(done => {
+    request = chai.request(server).get('/users');
+    done();
+  });
+
+  it(`should fail because ${sessionManager.HEADER_NAME} header is not being sent`, done => {
+    User.create({
+      name: 'Franco',
+      surname: 'Coronel',
+      email: 'franco.coronel@wolox.com.ar',
+      password: 'passwordFC'
+    });
+    request.catch(err => err.should.have.status(401));
```

Comment:
You are executing the usear creation and request simultaneously, is that what you want? Or you wanted to create the user and after that send the request?

Anyways, I think in this test you don't need any user.